Source/Core/Common/FileUtil.cpp

```diff
@@ -636,7 +636,7 @@ std::string GetCurrentDir()
   if (!(dir = __getcwd(nullptr, 0)))
   {
     ERROR_LOG(COMMON, "GetCurrentDirectory failed: %s", GetLastErrorMsg().c_str());
-    return nullptr;
+    return std::string();
```

Comment:
> yes can be "" or std::string( ), but "std::string = nullptr" allways crash program.

Yes, I know. What I'm saying is, is that it would be preferable if you could use `return "";` as it's less verbose.

> Or this intentionally crash??

It's likely a remnant before much of Common used std::string.